style.css

```
@@ -0,0 +1,89 @@
+body {
+    background: beige;
+    border: 4px ridge black;
+    box-shadow: 0 0 20px grey;
+}
+header{
+    color: brown;
+    background: burlywood;
+    font-family: "Franklin Gothic Heavy";
```

А если у пользователя нет такого шрифта? Следует хотя бы добавить в конце списком через запятую шрифт приблизительного начертания (с засечками, или без). в данном случае следует сделать примерно так:
```suggestion
    font-family: "Franklin Gothic Heavy", sans-serif;
```